buildroot/share/PlatformIO/variants/BIGTREE_SKR_PRO_1v1/variant.h

```diff
@@ -246,11 +246,11 @@ extern "C" {
 
 // Timer Definitions
 //Do not use timer used by PWM pins when possible. See PinMap_PWM in PeripheralPins.c
-#define TIMER_TONE              TIM6
+#define TIMER_TONE              TIM2
 #define TIMER_SERIAL            TIM7
 
 // Do not use basic timer: OC is required
-#define TIMER_SERVO             TIM2  //TODO: advanced-control timers don't work
+#define TIMER_SERVO             TIM6  //TODO: advanced-control timers don't work
```

Comment:
I am confused by this. TIM6 is a basic timer, and TIM2 is a General-purpose timer.
This contradicts the command in the line above. Do you know why this is working using the _less functional_ timer?

Reply:
Hi all,
the comment is a legacy one. Since HardwareTimer library has been deployed, this comment is no more relevant.
Using TIM6 is a good option like it is not linked to a pin.

Reply:
In that case we may need to check whether TIM2 is attached to any pins that might cause problems for the TONE_TIMER. Perhaps you are using some of the pins which attach to TIM2 as PWM outputs, and those are interfering with the timer operation.

(Sounds like the comment should be removed as well)